arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -3751,6 +3951,38 @@
       },
       "description": "The content hash."
     },
+    "RetryHistory": {
+      "type": "object",
+      "properties": {
+        "startTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets the start time."
```

Comment:
IMO, with or without "Gets", the descriptions in general can be made better. In the current state the description adds no further value since this part is already in the property name. But something like "The start time when the workflow recurrence would be triggered" would add more value.